.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
+  - 7.0
+  - 7.1
 
 env:
-  - SYMFONY_VERSION=2.6.*
-  - SYMFONY_VERSION="dev-master symfony/debug:~2.7@dev symfony/http-kernel:~2.7@dev"
+  - SYMFONY_VERSION=3.4.*
+  - SYMFONY_VERSION="dev-master symfony/debug:~3.4@dev symfony/http-kernel:~3.4@dev"
```

Comment:
add a matrix for multiple builds? See https://github.com/hostnet/form-handler-bundle/blob/master/.travis.yml#L15-L30